x-pack/plugins/reporting/export_types/common/layouts/print.css

```diff
@@ -19,7 +19,9 @@
 kbn-top-nav,
 .kbnToaster__container,
 filter-bar,
-::-webkit-scrollbar {
+::-webkit-scrollbar,
+.euiHeader,
```

Comment:
I think this can be removed since it's already addressed by https://github.com/elastic/kibana/pull/30000